Export Analysis Reports/Sdl.Community.ExportAnalysisReports/Helpers/Help.cs

```diff
@@ -125,7 +140,7 @@ public static void LoadReports(XmlDocument doc, string projectPath, ProjectDetai
 			}
 		}
 
-		public static bool ReportFileExist(string reportFolderPath)
+		public bool ReportFileExist(string reportFolderPath)
 		{
 			try
 			{
```

Comment:
At line 150: should use directly the return value of Any.
What if the user moved/deleted his files? Shouldn't the message also be displayed for this case?

Reply:
That's true, I've updated code to display the message also in that case and I've also refactored that method (remove redundant code)